tests/prod.yml.template.patch

```diff
@@ -0,0 +1,32 @@
+--- /Users/merlin/master/integration/production/config/prod.yml.template	2020-01-14 16:45:53.000000000 +0100
++++ production/config/prod.yml.template	2020-01-16 07:46:53.000000000 +0100
```

Comment:
So what is the problem in leaving these volumes mounted in the production environment?

In principle, we shouldn't apply this patch and try always to test production envs as close as possible to real production envs.